andela/temari-cfh
# Feature/137091007/users should be able to create or start a new game

**What does this PR do?**

Allows users to create and start new game

**Description of Task to be completed?**

Users should be able to start a new game session after login or signup.
api end points for post and put request for games should be created

**How should this be manually tested?**

Click on play with friends or play with strangers, a dialog box will appear asking you to continue, click on continue.

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

#137091007

**Screenshot**

<img width="1280" alt="screen shot 2017-02-13 at 1 06 14 am" src="https://cloud.githubusercontent.com/assets/24937708/22867443/af4d7c74-f188-11e6-92d9-4d1c37298edb.png">
